.github/workflows/pull_request.yml

```
@@ -121,15 +116,14 @@ jobs:
             USER_GID=${{ steps.user_ids.outputs.gid }}
             CPANMOPTS=--with-develop
           # Multi-layer caching strategy for maximum performance
+          # Note: we don't use gha cache because we run out of cache otherwise
+          # So we use registry cache, see https://docs.docker.com/build/cache/backends/registry/
           cache-from: |
-            type=local,src=/tmp/.buildx-cache
-            type=gha
+            type=registry,ref=ghcr.io/openfoodfacts-server/backend-buildx-cache:${{ hashFiles('Dockerfile*', 'docker-compose.yml', 'docker/**', 'cpanfile*', 'conf/apache*') }}
           cache-to: |
-            type=local,dest=/tmp/.buildx-cache-new,mode=max
-            type=gha,mode=max
+            type=registry,ref=ghcr.io/{{ github.repository }}/backend-buildx-cache:${{ hashFiles('Dockerfile*', 'docker-compose.yml', 'docker/**', 'cpanfile*', 'conf/apache*') }},mode=max
```

Shouldn't this be the same as line 122? Also missing a $ if you want to use a variable
```suggestion
            type=registry,ref=ghcr.io/openfoodfacts-server/backend-buildx-cache:${{ hashFiles('Dockerfile*', 'docker-compose.yml', 'docker/**', 'cpanfile*', 'conf/apache*') }},mode=max
```

---

yep, sorry I spotted it just after, it's fixed.